client/renderers/EGL/desktop.c

```diff
@@ -35,6 +35,9 @@ Place, Suite 330, Boston, MA 02111-1307 USA
 #include "desktop_rgb.frag.h"
 #include "desktop_rgh.def.h"
 
+#define LG_SCALE_AUTO    0
+#define SCALE_ALGORITHMS 3
```

Comment:
Instead of splitting these out here, please define them in `desktop_rgb.def.h`, and if possible use an enum with a `MAX` element, ie:
```c
enum EGLScaleAlgo
{
  EGL_SCALE_AUTO,
  EGL_SCALE_NEAREST,
  EGL_SCALE_LINEAR,

  EGL_SCALE_MAX
};
```

Also as these defines are local only to EGL, so please use `EGL_` for them

Reply:
Okay, defined the `enum`. Unfortunately, it cannot be defined in that header since it's generated from the shader, and you can't define enums in GLSL. Therefore, we have to manually keep the shader constants in sync from now on.

Reply:
It would certainly be better to keep them defined in one place, which is why I stated, "if possible". So yes, keep defines but keep them all in the one place with a `_MAX` define.

Reply:
Done.